app/assets/stylesheets/govuk_publishing_components/components/_image-card.scss

```diff
@@ -97,58 +100,58 @@
   }
 
   &.gem-c-image-card__list--indented {
-    padding-left: $gutter-half;
+    padding-left: govuk-spacing(3);
 
     .gem-c-image-card__list-item {
       position: relative;
 
       &:before {
         content: "-";
         position: absolute;
-        left: -$gutter-half;
+        left: (- govuk-spacing(3));
       }
     }
   }
 }
 
 .gem-c-image-card--large.gem-c-image-card {
-  margin: 0 0 30px 0;
+  margin: 0 0 govuk-spacing(6) 0;
 }
 
 .gem-c-image-card--large {
   .gem-c-image-card__image-wrapper {
-    padding: 0 $gutter-one-third 0 0;
-    margin: 0;
-    float: left;
-    max-width: 66.66%;
-  }
+    @include govuk-grid-column(two-thirds, $class: false, $at: tablet);
 
-  .gem-c-image-card__text-wrapper {
-    overflow: hidden;
-  }
+    margin-bottom: govuk-spacing(2);
+    float: none;
+    width: auto;
+    max-width: 100%;
+    padding: 0;
```

Comment:
`padding: 0` is being declared both here and on line 140 for I think the same element? Is this necessary?

Reply:
One is on `gem-c-image-card__image-wrapper` the other on `gem-c-image-card__text-wrapper`, two different elements I'm afraid.